cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -381,7 +376,7 @@ Slice::JavaGenerator::fixKwd(const string& name)
 }
 
```

Comment:
Everything below this line is just me switching from `<function> const` to `static <function>`.
There is no reason for these not to be static, but it just happens that I _needed_ them to be static in this PR, so I could call them from outside the class they're defined in.